### ISSUE TITLE
The mask-rcnn structure graph is the same as faster-rcnn?

### ISSUE THREAD

**Issue by lilichu** [facebookresearch/Detectron#614]
hi!
I  output the structure graph of mask-rcnn, but I find it is the same as the structure graph of faster-rcnn.
 
The net such as the following:
```
INFO net.py: 250: _[mask]_roi_feat_shuffled   : (13, 256, 14, 14)    => _[mask]_roi_feat            : (13, 256, 14, 14)    ------- (op: BatchPermutation)
INFO net.py: 250: mask_rois_idx_restore_int32 : (13,)                => _[mask]_roi_feat            : (13, 256, 14, 14)    ------|
INFO net.py: 250: _[mask]_roi_feat            : (13, 256, 14, 14)    => _[mask]_fcn1                : (13, 256, 14, 14)    ------- (op: Conv)
INFO net.py: 250: _[mask]_fcn1                : (13, 256, 14, 14)    => _[mask]_fcn1                : (13, 256, 14, 14)    ------- (op: Relu)
INFO net.py: 250: _[mask]_fcn1                : (13, 256, 14, 14)    => _[mask]_fcn2                : (13, 256, 14, 14)    ------- (op: Conv)
INFO net.py: 250: _[mask]_fcn2                : (13, 256, 14, 14)    => _[mask]_fcn2                : (13, 256, 14, 14)    ------- (op: Relu)
INFO net.py: 250: _[mask]_fcn2                : (13, 256, 14, 14)    => _[mask]_fcn3                : (13, 256, 14, 14)    ------- (op: Conv)
INFO net.py: 250: _[mask]_fcn3                : (13, 256, 14, 14)    => _[mask]_fcn3                : (13, 256, 14, 14)    ------- (op: Relu)
INFO net.py: 250: _[mask]_fcn3                : (13, 256, 14, 14)    => _[mask]_fcn4                : (13, 256, 14, 14)    ------- (op: Conv)
INFO net.py: 250: _[mask]_fcn4                : (13, 256, 14, 14)    => _[mask]_fcn4                : (13, 256, 14, 14)    ------- (op: Relu)
INFO net.py: 250: _[mask]_fcn4                : (13, 256, 14, 14)    => conv5_mask                  : (13, 256, 28, 28)    ------- (op: ConvTranspose)
INFO net.py: 250: conv5_mask                  : (13, 256, 28, 28)    => conv5_mask                  : (13, 256, 28, 28)    ------- (op: Relu)
INFO net.py: 250: conv5_mask                  : (13, 256, 28, 28)    => mask_fcn_logits             : (13, 13, 28, 28)     ------- (op: Conv)
INFO net.py: 250: mask_fcn_logits             : (13, 13, 28, 28)     => loss_mask                   : ()                   ------- (op: SigmoidCrossEntropyLoss)
INFO net.py: 250: masks_int32                 : (13, 10192)          => loss_mask                   : ()                   ------|

```
After` roi_feat`, it should be `fcn1`, `fcn2`, `fcn3`, `fcn4`...  But why does the output of structure graph not contain these? 
The  structure graph of mask-rcnn is such as:
![image](https://user-images.githubusercontent.com/28217651/44011925-319667ae-9eee-11e8-994c-1b9ad07436b1.png)
the graph is the same as faster-rcnn. Thanks!
